### PR TITLE
update webapp-install-banner.js helpText

### PR DESCRIPTION
**Pull request by kaycebasques**

### CODE REVIEW

File: lighthouse-core/audits/webapp-install-banner.js [hunk 1]
```diff
@@ -38,7 +38,9 @@ class WebappInstallBanner extends MultiCheckAudit {
       category: 'PWA',
       name: 'webapp-install-banner',
       description: 'User can be prompted to Install the Web App',
-      helpText: 'While users can manually add your site to their homescreen, the [prompt (aka app install banner)](https://developers.google.com/web/fundamentals/engage-and-retain/app-install-banners/) will proactively prompt the user to install the app if the various requirements are met and the user has moderate engagement with your site.',
+      helpText: 'Browsers can proactively prompt users to add your app to their homescreen. ' +
+          'Users who add apps to their homescreen engage those apps more. ' +
```
Comment (patrickhulce) [inline lighthouse-core/audits/webapp-install-banner.js]: "engage with those apps more" maybe?

Comment (paulirish) [inline lighthouse-core/audits/webapp-install-banner.js]: Or.. maybe something like.

A homescreen icon leads to higher engagement.

Comment (kaycebasques) [inline lighthouse-core/audits/webapp-install-banner.js]: How about:

Browsers can proactively prompt users to add your app to their homescreen, which can lead to higher engagement.

**Comment by kaycebasques** [comment]
henlo @patrickhulce and @paulirish my ol frens, plz look at updated text

**Pull request merged**